db/models.py

```diff
@@ -1 +1,39 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(
+        verbose_name="This field describes what kind of"
+                     " bonus players can get from it.",
+        max_length=255
+    )  # дописать
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255, unique=False)
+    bio = models.CharField(
+        verbose_name="It stores a short description"
+                     " provided by a user about himself/herself.",
+        max_length=255
+    )
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+    guild = models.ForeignKey(
+        Guild,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL
+    )
+    created_at = models.DateTimeField(auto_now=True)
```

Comment:
Consider using `auto_now_add=True` instead of `auto_now=True` for the 'created_at' field to ensure it only sets the timestamp when the object is first created.